bpf/bpf_network.c

```diff
@@ -19,8 +19,7 @@ int from_network(struct __ctx_buff *ctx)
 
 	if ((ctx->mark & MARK_MAGIC_HOST_MASK) == MARK_MAGIC_DECRYPT) {
 		send_trace_notify(ctx, TRACE_FROM_NETWORK, get_identity(ctx), 0, 0,
-				  ctx->ingress_ifindex,
-				  TRACE_REASON_ENCRYPTED, TRACE_PAYLOAD_LEN);
+				  ctx->ingress_ifindex, 0, TRACE_PAYLOAD_LEN);
```

Comment:
I think we have [the same issue at `bpf_overlay`](https://github.com/cilium/cilium/blob/v1.11.1/bpf/bpf_overlay.c#L306-L308).

Reply:
Ok, let me fix that together.